cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -195,6 +167,15 @@ def toggle_ingress_state():
     remove_state('kubernetes-worker.ingress.available')
 
 
+@when('docker.sdn.configured')
+def sdn_changed():
```

Comment:
The configuration blobs were removed and replaced by this succinct method. :+1: good job here!